packages/google_mobile_maps/android/src/main/java/io/flutter/plugins/googlemobilemaps/GoogleMobileMapsPlugin.java

```diff
@@ -182,6 +198,10 @@ private static LatLngBounds toLatLngBounds(Object o) {
     return new LatLngBounds(toLatLng(data.get(0)), toLatLng(data.get(1)));
   }
 
+  private static boolean toBoolean(Object o) {
```

Comment:
neccesary?

Reply:
Made use of `toXxx(...)` consistent across all types.